packages/react-devtools-shared/src/__tests__/utils.js

```diff
@@ -194,18 +194,20 @@ export function exportImportHelper(bridge: FrontendBridge, store: Store): void {
   );
   const parsedProfilingDataExport = JSON.parse(serializedProfilingDataExport);
 
-  const profilingDataFrontend = prepareProfilingDataFrontendFromExport(
+  const profilingDataFrontendImport = prepareProfilingDataFrontendForImport(
```

Comment:
This is still wrong, at least in my mental model. The data that's been prepared isn't an import anymore. It's just "profiling data", as can be seen below when it's assigned to
```js
profilerStore.profilingData = profilingDataFrontendImport;
```

I think these renames confuse me because they go against my mental model of this stuff.